dev-guide/index.md

```diff
@@ -0,0 +1,331 @@
+
+### Your Profile
+
+Take some time to fill out your profile. This is a chance to tell people a little about yourself and your goals as a developer. Maybe talk a bit about what kinds of games interest you, if you're looking for collaborators, if you're into game jams, and that kind of thing. If you have a Patreon page or other personal websites, feel free to put links here. 
+
+Speaking of links, linking your social media accounts with Game Jolt is more important than you might think. Don't put your Twitter and Facebook links in the body of your profile; [link them](http://gamejolt.com/dashboard/linked-accounts/) directly. They will appear big and bold on the side of your profile, which makes it easier for fans, press, staff, etc. to contact you.
+
+* * *
+
+## Add a Game
+
+Dev account all set up? Good. Now you're ready to start adding your games to the site.
+
+#### What kind of games can be uploaded?
+
+Game Jolt is currently focused exclusively on free indie games. These can be hobbyist creations, labors of love, early prototypes, jam games, student projects, fan games, experiments, art games, alt games, or of any other origin--as long as they are free.
+
+They can be open source, Creative Commons licensed, or all rights reserved; they just have to be uploaded by their developer and they have to be free. Publishers cannot upload games; only developers. 
+
+#### Adding a Game
+
+Go ahead and hit "Dashboard" on the black nav bar. Welcome to your developer dashboard! This is where you can access any games you have added. Now hit the big "Add a Game" button. Go ahead, do it.
+
+### The Rules
+
+The first thing you'll see are the rules for adding a game to Game Jolt. The rules are there for a reason. You have to agree to them before you add a game, and if your game breaks the rules, it will be removed. Read the rules and abide by them. That way everyone stays happy and nobody gets hurt. That last bit was a joke. Someone always gets hurt. Just kidding. The rules are serious, though.
+
+### Game Information
+
+You'll need to fill in some basic information about your game in order to proceed.
+
+**Title:** Surely your game has a name. Put it here.
+
+- *A note on mature language:* If your game's title contains the "F-word" or a swear word of that magnitude, please replace one or more of its letters with *. This will keep browsing friendly for people who do not wish to view mature content. This only applies to titles and thumbnails because they show up in lists and search results; once someone visits a game page, there is a maturity rating on display and one can make their own decision about whether to read the description and play the game. 
```

Comment:
Cancel out the * by either putting it in backticks, or by putting a backslash before. It currently is trying to make it italic.